### PR TITLE
Fixing the "relogIssues" build config option.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

**Comment by nickpape-msft** [comment]
Your title and changefile have a weird spelling error: "reolgIssues"

**Comment by nickpape-msft** [comment]
Add a description

File: core-build/gulp-core-build/src/logging.ts [hunk 1]
```diff
@@ -183,6 +182,7 @@ function writeSummary(callback: () => void): void {
 
     // flush the log
     afterStreamsFlushed(() => {
+      const shouldRelogIssues: boolean = getFlagValue('relogIssues');
```
Comment (nickpape-msft) [inline core-build/gulp-core-build/src/logging.ts]: >const shouldRelogIssues: boolean = getFlagValue('relogIssues'); [](start = 6, length = 63)

doesn't it already re-log all the issues without this flag?

File: core-build/gulp-core-build/src/logging.ts [hunk 2]
```diff
@@ -174,7 +174,6 @@ function afterStreamsFlushed(callback: () => void): void {
 }
 
 function writeSummary(callback: () => void): void {
-  const shouldRelogIssues: boolean = getFlagValue('relogIssues');
```
Comment (nickpape-msft) [inline core-build/gulp-core-build/src/logging.ts]: >getFlagValue [](start = 37, length = 12)

why are we looking at getFlagValue. Shouldn't it be getConfigValue ?

File: core-build/gulp-core-build/src/logging.ts (hunk 1 above)
Comment (iclanton) [reply core-build/gulp-core-build/src/logging.ts]: The default value for that flag is "true"

File: core-build/gulp-core-build/src/logging.ts (hunk 2 above)
Comment (iclanton) [reply core-build/gulp-core-build/src/logging.ts]: getFlagValue looks at the command line, the environment, and at the build config.

**Pull request merged**